src/components/common/Directors.vue

```diff
@@ -736,30 +696,25 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
    * NB: Do not validate inter word spacing because the Legal db needs to support
    *     such records in order to correctly update Colin.
    */
-  private readonly directorLastNameRules: Array<Function> = [
+  private readonly directorLastNameRules: Function[] = [
     v => !!v || 'A last name is required',
     v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
     v => !/\s$/g.test(v) || 'Invalid spaces' // trailing spaces
   ]
 
-  /** JSON representation of directors list for detecting array size and property changes. */
-  private get directorsJson (): string {
```

Comment:
This was to be able to identify changes to arrays/sub-objects. This is not needed if you do a deep watch, so I changed it.